docs/install/installing-istio.md

```diff
@@ -139,6 +139,101 @@ spec:
 EOF
 ```
 
+After you install the cluster local gateway, your service and deployment for the local gateway are both named `cluster-local-gateway`.
+You do not need to update the `config-istio` configmap under the `knative-serving` namespace, because Knative Serving can use
```

Comment:
This sentence, while I assume is true, feels a bit out of place because it assumes the reader wanted to update the configMap - but why would they? A newbie wouldn't have even thought about doing this so I don't think there's any need to mention it. I think you can remove it.